spec/spec_helper.rb

```diff
@@ -0,0 +1,31 @@
+require 'simplecov'
+require 'simplecov-lcov'
```

Comment:
You can move this simplecov config to `.simplecov` in root dir.

Reply:
Decided not to move it, because it caused running coverage while db setup and failing build. We could add to gemfile `require: false` for simplecov but  then undercover would fail because simplecov-lcov requires simplecov(https://stackoverflow.com/questions/32751359/why-does-simplecov-trigger-when-rake-dbschemaload-is-executed)

Reply:
Nope :)